docs/developer/release-team.md

```diff
@@ -0,0 +1,70 @@
+# Cluster API Release Team
+
+## Overview
+
+In the past, releasing Cluster API has been mostly ad-hoc and relied on one or more contributors to do most of the chore work necessary to prepare the release. One of the major downsides of this approach is that it is often difficult for users and providers to plan around Cluster API releases as they often have little visibility around when a release will happen. 
+
+This document introduces the concept of a release team with the following goals and non-goals:
+
+### Goals
+
+- To improve communication to end users and contributors about CAPI release cadence and target release dates
+- To spread the load of release tasks and involve a bigger, more diverse set of CAPI contributors in cutting releases
+- To look at the Kubernetes SIG release team processes for guidance on releasing best practices
+- To improve tooling, documentation, and automation of the CAPI release process
+
+### Non-Goals/Future work
+
+- To increase the frequency of releases (AKA release cadence). This will be revisited in the future once the release process stabilizes.
+- To change the current proposal (CAEP) process
+- To copy implement all steps of the Kubernetes release process for CAPI releases
+
+Note that this document is intended to be a starting point for the release team. It is not a complete release process document.
+
+More details on the CAPI release process can be found in [this past issue tracking release tasks](https://github.com/kubernetes-sigs/cluster-api/issues/6615) and the current [release documentation](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/developer/releasing.md).
+
+## Duration of Term
+
+Each release team term will last for a quarter, to align with one minor release cycle. A minor release cycle starts right after a minor release and concludes with the release of the next minor release.
```

Comment:
Also, what about aiming at three releases/four months cycle per year for starting?
It will be already a great improvement from what we have now with a little bit less pressure on the team/a smmother transition from the current 2 releases per year we historically managed to create (and it also opens up to align our candence with Kubernetes cadence)

Reply:
Changed to four-month cycle, the google doc had quarterly in one place and four months in another